scripts/main.js

```diff
@@ -0,0 +1,78 @@
+const bookCard = document.getElementById('book-card')
+
+const myLibrary = []
+
+function Book(author, title, pages, read = false) {
+    this.id = Math.floor(Math.random() * 100)
```

Comment:
[OPTIONAL]
Consider changing this to more controlled. (There is still possibility of getting the same number twice, which does not make it unique)